### PR TITLE
Update communication.md

### PR DESCRIPTION
**Pull request by msbemba**
Fixed typo. It should be Channel instead of Chat.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3521

File: microsoft-365/admin/productivity/communication.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ We also provide you with information that help you gain visibility into how peop
       - **People sending chat messages:** The colored portion and the fraction represent the use of chat messages within people who sent messages on Microsoft Teams. The fraction is constructed from:
         - **Numerator:** People who sent chat messages on Microsoft Teams in the last 28 days
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/productivity/communication.md]:
```suggestion
        - **Numerator:** People who sent chat messages on Microsoft Teams in the last 28 days.
```

File: microsoft-365/admin/productivity/communication.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ We also provide you with information that help you gain visibility into how peop
...
         - **Denominator:** People sent messages on Microsoft Teams in the last 28 days
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/productivity/communication.md]:
```suggestion
        - **Denominator:** People who sent messages on Microsoft Teams in the last 28 days.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few additional changes, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**